bindings/kitsune_runner/src/definition.rs

```diff
@@ -21,6 +21,6 @@ impl<RV: UserValuesConstraint, AV: UserValuesConstraint> KitsuneScenarioDefiniti
     /// Once the Kitsune customisations have been made, use this function to switch back to
     /// configuring default properties for the scenario.
     pub fn into_std(self) -> ScenarioDefinitionBuilder<RV, AV> {
-        self.inner
+        self.inner.add_capture_env("NUM_MESSAGES")
```

Comment:
Similarly this forces every scenario to use this. Maybe that's okay but I'd like to check

Reply:
https://github.com/holochain/wind-tunnel/pull/153/commits/b9e7882a5918c439791e7a9ab156f65c35b10cf0